common/v2/features/SendAssets/stateFactory.tsx

```diff
@@ -56,22 +71,85 @@ const TxConfigFactory: TUseApiFactory<State> = ({ state, setState }) => {
   };
 
   // For Other Wallets
-  // @ts-ignore
-  const handleConfirmAndSend: TStepAction = (payload, after) => {};
+  // tslint:disable-next-line
+  const handleConfirmAndSend: TStepAction = (_payload, after) => {
+    const provider = new ProviderHandler(state.txConfig.network);
+    if (state.signedTx) {
+      provider
+        .sendRawTx(state.signedTx)
+        .then(transactionReceipt => {
+          setState((prevState: State) => ({
+            ...prevState,
+            txReceipt: transactionReceipt
+          }));
+          after();
+        })
+        .catch(txHash => {
+          // If rejected, data is a tx hash, not a receipt. Fetch the receipt, then save receipt for flow
+          provider.getTransactionByHash(txHash).then(transactionReceipt => {
+            setState((prevState: State) => ({
+              ...prevState,
+              txReceipt: transactionReceipt
+            }));
+            after();
+          });
+        });
+    }
+  };
 
-  // @ts-ignore
-  const handleSignedTx: TStepAction = (payload, after) => {};
+  const handleSignedTx: TStepAction = (payload: ISignedTx, after) => {
+    const decodedTx = decodeTransaction(payload);
+    const networkDetected = getNetworkByChainId(decodedTx.chainId);
+    const contractAsset = getAssetByContractAndNetwork(decodedTx.to || undefined, networkDetected);
+    setState((prevState: State) => ({
+      ...prevState,
+      signedTx: payload,
+      txConfig: {
+        rawTransaction: prevState.txConfig.rawTransaction,
+        receiverAddress: contractAsset ? decodeTransfer(decodedTx.data)._to : decodedTx.to,
+        amount: contractAsset
+          ? fromTokenBase(
+              toWei(decodeTransfer(decodedTx.data)._value, 0),
+              contractAsset.decimal || 18
+            )
+          : decodedTx.value,
+        network: networkDetected || state.txConfig.network,
```

Comment:
react state functions are async, which means that we can't be sure what the values of `state` actually reference during the update (could be the result of another state update that occurred after this `setState` was triggered).
So prefer using `prevState` instead (for the entire method)

Reply:
done.